chalicelib/checks/wfr_checks.py

```diff
@@ -247,12 +249,30 @@ def line_count_test(connection, **kwargs):
             action.description = 'Did not complete action due to time limitations'
             break
         try:
-            linecount_result = check_lines(metawfr_uuid, my_auth, steps=steps_dict, fastqs=fastqs_dict)
             metawfr_meta = ff_utils.get_metadata(metawfr_uuid, add_on='?frame=raw', key=my_auth)
+            # we have a few different dictionaries of steps to check output from in linecount_dicts.py
+            # the proband-only and family workflows have the same steps, so we assign the proband_SNV_dict
+            if 'Proband-only' in metawfr_meta['title'] or 'Family' in metawfr_meta['title']:
+                steps_dict = proband_SNV_dict
+            # trio has novoCaller, so it has a separate dictionary of steps
+            elif 'Trio' in metawfr_meta['title']:
+                steps_dict = trio_SNV_dict
+            # cnv/sv is a completely different pipeline, so has a many different steps
+            elif 'CNV' in metawfr_meta['title']:
+                steps_dict = CNV_dict
+            # if this is run on something other than those expected MWFRs, we want an error.
+            else:
+                e = 'Unexpected MWF Title: '+metawfr_meta['title']
+                action_logs['error'] = str(e)
+                raise Exception(str(e))
```

Comment:
I'm not sure you want to raise an Exception here, this would prevent the action from returning and we wouldn't be able to see the logs. The line above for adding to the action_logs is probably enough.

Reply:
removed the exception - thanks!